xml/System/HashCode.xml

```diff
@@ -435,10 +479,9 @@
         <Parameter Name="obj" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="obj">Ignored.</param>
+        <summary>This method is not supported and should not be called.</summary>
+        <returns>This method will always throw a <see cref="System.NotSupportedException" />.</returns>
```

Comment:
Not sure how to cref exceptions in this format.

Reply:
Use the xref syntax:

```xml
<xref:System.NotSupportedException?displayProperty=nameWithType>
```

Reply:
I think they looked correct before except they were missing the T:

Reply:
Yep, I figured it out by grepping the existing docs.

Reply:
Good. We'll review the PR soon. Thanks!